### ISSUE TITLE
antd 5 is not working with @react-querybuilder/antd 

### ISSUE THREAD

**Issue by kimhandole** [react-querybuilder/react-querybuilder#573]
Only antd 4 is working, not antd 5.
Would some one please update @react-querybuilder/antd so it can be worked with antd 5 please?

**Comment by jakeboone02** [comment]
It should be compatible with `antd@5`. Could you set up an example repo demonstrating the issues you're encountering? A fork-able example project can be found here: https://github.com/react-querybuilder/react-querybuilder/tree/main/examples/antd.

**Comment by jakeboone02** [comment]
antd v5 has been supported since [RQB v5.1.3](https://github.com/react-querybuilder/react-querybuilder/blob/main/CHANGELOG.md#v513---2022-11-23), which was released almost a year ago. Original PR is #387.

**Comment by kimhandole** [comment]
Hi @jakeboone02, 
Thank you for the fast response I really appreciate it!

Below is the example that has the error (which is just code example in React Query Builder document site, the only difference is that I upgraded antd version to "5.3.0" from package.json file)
example: https://stackblitz.com/edit/github-afmfqz?file=package.json

I tried on my project and had exactly same error. The error is below:
TypeError: date_picker_default.generatePicker is not a function
    at @react-querybuilder_antd.js?v=83995a16:99864:39
    
Please let me know how it goes, can't wait to use this awesome library!

**Comment by kimhandole** [comment]
Sorry, closed the issue by accident! 
Reopend!

**Comment by jakeboone02** [comment]
If you pin the two RQB packages to "6.5.0" (no leading caret), your StackBlitz works.

[Updated, working version here](https://stackblitz.com/edit/github-afmfqz-qn2ckr?file=package.json).

```jsonc
{
  "dependencies": {
    "@react-querybuilder/antd": "6.5.0",
    // ...
    "react-querybuilder": "6.5.0"
  }
  // ...
}
```

I'll see if I can patch v6.5.2 to work properly as well.

**Comment by kimhandole** [comment]
I was able to use it in my project, thank you so much for the answer!
Okay, I really appreciate it!!

**Comment by jakeboone02** [comment]
Reopening this since it's broken in the latest release.

**Comment by jakeboone02** [comment]
@kimhandole - This issue was closed automatically when I merged PR #585, but I do think it's actually resolved. I've released a new v7 alpha, [7.0.0-alpha.1](https://www.npmjs.com/package/react-querybuilder/v/7.0.0-alpha.1), which includes the fix. Can you give it a try and see if you don't get the error message? v7 shouldn't require many changes, if any, to upgrade from v6.

If it works then I'll release v6.5.4 with the same changes so you won't have to wait on v7 final.

**Comment by xianghongai** [comment]
@jakeboone02 hi bigwig, How to adjust the size of button and input? like `size="small" `
